api/system-backup/read

```diff
@@ -104,6 +104,28 @@ sub find_latest_log
     return ($latest_filename, $latest_ts);
 }
 
+sub find_all_logs
+{
+    my $name = shift;
+    my $n = "[0123456789]";
+    my $logs;
+    foreach my $file (glob("/var/log/backup/backup-$name-$n$n$n$n$n$n$n$n$n$n$n$n.log")) {
```

Comment:
```suggestion
    foreach my $file (glob("/var/log/backup/backup-$name-$n.log")) {
```
While exact digits count may be more accurate, I think it's safe enough getting all digits.
Removing $n (remove also line 110) should improve clarity.

Reply:
I'm not sure this change works :thinking: because it's not a regex, let me try. However I copied the previous method which did the same :grin:

Reply:
Proposed match will not work, you need to use:
```
/var/log/backup/backup-$name-$n*.log
```
Still, I prefer the current implementation: it will match exactly needed files.